src/js/components/projects/createForm.tsx

```diff
@@ -188,126 +190,182 @@ const ProjectForm = ({ user, repository, hasProjects, history }: Props) => {
       </>
     );
   }
-
+  // "create and new" submit button events
+  const createSuccess = (action: AnyAction) => {
+    const {
+      type,
+      payload: { object, objectType },
+    } = action;
+    setIsSaving(false);
+    closeForm();
+    if (
+      type === 'CREATE_OBJECT_SUCCEEDED' &&
+      objectType === OBJECT_TYPES.PROJECT &&
+      object?.slug
+    ) {
+      const url = routes.project_detail(repository.slug, object.slug);
+      history.push(url);
+    }
+  };
+  const submitClicked = (e) => {
+    if (inputs.name) {
+      setIsSaving(true);
+      handleSubmit(e, undefined, (action) => createSuccess(action));
```

Comment:
This feels icky, we should probably use optional parameters (aka named params) for functions that have more than 2 arguments https://2ality.com/2011/11/keyword-parameters.html#:~:text=JavaScript%20does%20not%20have%20native,to%20a%20single%20formal%20parameter.